spring-integration-azure/spring-integration-eventhub/src/test/java/com/microsoft/azure/spring/integration/eventhub/EventHubOperationSendSubscribeTest.java

```diff
@@ -63,7 +64,8 @@ protected void verifyCheckpointFailureCalled(int times) {
 
     @Test
     public void testSendReceiveWithBatchCheckpointMode() {
-        sendSubscribeOperation.setCheckpointMode(CheckpointMode.BATCH);
+        sendSubscribeOperation.setCheckpointConfig(CheckpointConfig.builder().checkpointMode(CheckpointMode.BATCH).build
+                ());
```

Comment:
nit: format?

Reply:
what do you mean by this?

Reply:
brackets should be put together with the method call?

Reply:
it's in the same line, the line break is caused by github view

Reply:
from my experience, put the PR review window to the biggest monitor...